apps/console/src/features/applications/pages/applications.tsx

```diff
@@ -278,6 +334,138 @@ const ApplicationsPage: FunctionComponent<ApplicationsPageInterface> = (
         return <RemoteFetchStatus data-testid={ "remote-fetch" } />;
     };
 
+    /**
+     * Handles the My Account Portal status update action.
+     */
+     const handleMyAccountStatusToggle = (e: SyntheticEvent, data: CheckboxProps) => {
```

Comment:
Add TSDoc comments as well for params and return.
```suggestion
     const handleMyAccountStatusToggle = (e: SyntheticEvent, data: CheckboxProps): void => {
```

Reply:
Addressed with 06eac07bbcb1c736313953e0cc52b62e0ac383a7